radixdlt/src/main/java/com/radixdlt/CerberusModule.java

```diff
@@ -72,10 +76,12 @@ public CerberusModule(RuntimeProperties runtimeProperties) {
 	protected void configure() {
 		// dependencies
 		bind(Scheduler.class).toProvider(SingleThreadedScheduler::new);
-		bind(PacemakerRx.class).to(PacemakerImpl.class);
-		bind(Pacemaker.class).to(PacemakerImpl.class);
+		bind(TimeoutSender.class).to(ScheduledTimeoutSender.class);
+		bind(PacemakerRx.class).to(ScheduledTimeoutSender.class);
+		bind(Pacemaker.class).to(FixedTimeoutPacemaker.class);

```

Comment:
Configuration here seems to have been a bit messed up over time.  This binding should be removed.

See also FixedTimeoutPacemaker provider below, which should be a Pacemaker provider.